node/network/src/message.rs

```diff
@@ -1,57 +1,42 @@
+use serde_derive::{Deserialize, Serialize};
+
 use primitives::beacon::SignedBeaconBlock;
 use primitives::chain::{ChainPayload, ReceiptBlock, SignedShardBlock};
 use primitives::hash::CryptoHash;
-use primitives::types::{AccountId, BlockId, Gossip};
 use primitives::transaction::SignedTransaction;
-use serde_derive::{Deserialize, Serialize};
+use primitives::types::Gossip;
 
 pub type RequestId = u64;
```

Comment:
What is `RequestId`?

Reply:
it's request id for each client to account for their requests. just some increasing value for now.

Reply:
So it is a request nonce. Why do we need it at all?

Reply:
Id for client to distinguish responses for requests. auto increment for now.